lib/common_test/test/common_test_gh.spec

```diff
@@ -1,2 +1,3 @@
 {suites,"../common_test_test",all}.
 {skip_suites,"../common_test_test",[ct_release_test_SUITE],"Versions not always correct"}.
+{event_handler, {cte_track, []}}.
```

Comment:
`{enable_builtin_hooks, false}.` addition is not needed? no logger events messing the output?
maybe they're generated less frequently for common_test suites ...

Reply:
OK. maybe it is not relevant for ct.

Reply:
If we're adding this enable_builtin_hoost, shouldn't we also add it to common_test.spec as well?